xoops_trust_path/modules/xelfinder/sql/mysql.sql

```diff
@@ -17,7 +17,7 @@ CREATE TABLE `file` (
   `mime_filter` varchar(255) NOT NULL DEFAULT '',
   `local_path` varchar(255) NOT NULL DEFAULT '',
   PRIMARY KEY (`file_id`),
-  UNIQUE KEY `parent_name` (`parent_id`,`name`),
+  UNIQUE KEY `parent_name` (`parent_id`,`name`(100)),
```

Comment:
It is necessary to check for duplicate file names, so partial indexes cannot be adopted.
This will limit the file name to 191 characters.